src/DifferentThemeVariant.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Phpab\Phpab;
+
+class DifferentThemeVariant implements VariantInterface
```

Comment:
So this is a framework implementation right? This would not be part of the library itself?